### PR TITLE
Update TinSurfaceCreate.py

### PR DESCRIPTION
**Pull request by spono**
minor adjustments and some funtionalities added (first return selection, output as .asc)

### CODE REVIEW

File: python/plugins/processing/algs/lidar/fusion/TinSurfaceCreate.py [hunk 1]
```diff
@@ -81,3 +91,9 @@ def processAlgorithm(self, progress):
         else:
             commands.extend(files)
         FusionUtils.runFusion(commands, progress)
+        commands = [os.path.join(FusionUtils.FusionPath(), 'DTM2ASCII.exe')]
+        #commands.append('/raster')
```
Comment (michaelkirk) [inline python/plugins/processing/algs/lidar/fusion/TinSurfaceCreate.py]: accidental comment?

**Comment by michaelkirk** [comment]
Are there any tests for the processing plugins, and how do I run them?

File: python/plugins/processing/algs/lidar/fusion/TinSurfaceCreate.py (hunk 1 above)
Comment (spono) [inline python/plugins/processing/algs/lidar/fusion/TinSurfaceCreate.py]: sorry, "#" should be removed

**Comment by m-kuhn** [comment]
@michaelkirk not yet unfortunately, I'll be working on it soon

**Comment by michaelkirk** [comment]
I'm interested in working on testing in QGIS, so please point me to any
WIP.

**Comment by m-kuhn** [comment]
@michaelkirk I barely started https://github.com/m-kuhn/QGIS/tree/processing-tests

**Pull request merged**